mailpoet/assets/js/src/form_editor/form_preview.ts

```diff
@@ -21,12 +21,12 @@ jQuery(($) => {
         return;
       }
       // Allow message processing only when send from editor's origin
-      const editorUrl = new URL($previewForm.data('editor-url'));
+      const editorUrl = new URL(<string> $previewForm.data('editor-url'));
       if (editorUrl.origin !== event.origin) {
         return;
       }
 
-      const formType = event.data.formType;
+      const formType = <string> event.data.formType;
```

Comment:
I think we should stick to one syntax of type assertions and since we use `as` everywhere, it would be good to use it here as well.

Reply:
That makes sense. In general I prefer `<string>`, but it doesn't work in tsx files, this is why i switched in tsx files to `as string`. But makes sense to have it consistent.

Reply:
I think `<type>` is the older way to use it and the `as` was added later. Not sure why, maybe `<type>` can be confused with generics. Today, `as` seems to be more common to me.

Reply:
Addressed in 60f94f6ea